### PR TITLE
Added STUFF

### PR DESCRIPTION
**Pull request by absoIute**
HI MERGE PLEASE

### CODE REVIEW

File: bindings/2.200/GeometryDash-extras.bro [hunk 1]
```diff
@@ -1640,7 +1640,7 @@ class EndLevelLayer : GJDropDownLayer {
 	void onEveryplay(cocos2d::CCObject* sender);
 	void onLevelLeaderboard(cocos2d::CCObject* sender);
 	void onEdit(cocos2d::CCObject* sender);
-	void onMenu(cocos2d::CCObject* sender);
+	void onMenu(cocos2d::CCObject* sender) = 0xe81d0;
```
Comment (matcool) [inline bindings/2.200/GeometryDash-extras.bro:1643]: missed one

File: bindings/2.200/GeometryDash.bro [hunk 2]
```diff
@@ -1576,7 +1576,7 @@ class GameManager : GManager {
 	TodoReturn completedAchievement(gd::string);
 	TodoReturn defaultYOffsetForBG2(int);
 	TodoReturn generateSecretNumber();
-	TodoReturn getPracticeMusicFile();
+	gd::string getPracticeMusicFile() = 0x11f990;
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:1579]: another

**Comment by Cvolton** [comment]
bindings popping off

File: bindings/2.200/GeometryDash-extras.bro (hunk 1 above)
Comment (Junaidkassar) [reply bindings/2.200/GeometryDash-extras.bro]: Hi

**Pull request merged**